doc/themes/the-things-stack/layouts/partials/nav.html

```diff
@@ -39,3 +42,12 @@
     </div>
   </div>
 </nav>
+
+<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
```

Comment:
We need to make sure our Content Security Policy allows this domain for JS.

Reply:
where is the CSP?